docs/sdd.md

```diff
@@ -1,4 +1,4 @@
-# Software Design Document (SDD) - `svc-FIXME` 
+# Software Design Document (SDD) - `svc-gis` 
```

Comment:
Start of document format. Should start with the banner (and should not be restricted to width).

From svc-template-rust:
```
![Arrow Banner](https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png)

# Software Design Document (SDD) - `svc-FIXME`

## :telescope: Overview

This document details the software implementation of FIXME.

This service is responsible for FIXME

### Metadata

| Attribute     | Description                                                       |
| ------------- |-------------------------------------------------------------------|
| Maintainer(s) | [Services Team](https://github.com/orgs/Arrow-air/teams/services) |
| Stuckee       |  |
| Status        | Draft                                                             |

## :books: Related Documents

Document | Description
--- | ---
[High-Level Concept of Operations (CONOPS)](https://github.com/Arrow-air/se-services/blob/develop/docs/conops.md) | Overview of Arrow microservices.
[High-Level Interface Control Document (ICD)](https://github.com/Arrow-air/se-services/blob/develop/docs/icd.md)  | Interfaces and frameworks common to all Arrow microservices.
[Requirements - `svc-FIXME`](FIXME - generate a link specifically for this module's view in NocoDB) | Requirements and user stories for this microservice.
[Concept of Operations - `svc-FIXME`](./conops.md) | Defines the motivation and duties of this microservice.
[Interface Control Document (ICD) - `svc-FIXME`](./icd.md) | Defines the inputs and outputs of this microservice.
```

Reply:
https://github.com/Arrow-air/svc-gis/commit/2b93948539015c3003bde9fba7385a81e9e6f4dd